public/index.html

```diff
@@ -10,6 +10,7 @@
     -->
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json">
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
+    <script async defer src="https://buttons.github.io/buttons.js"></script>
```

Comment:
What does this do? :cat:

Reply:
And yes! Great idea with the header. I was saying to @titojankowski that it would be super nice if we could get some sort of "Carbon Doomsday" logo to replace this text.

Reply:
Yeah, logo - definitely, let's get that.